### DISCUSSION TITLE
Android settings

### DISCUSSION THREAD

**Discussion by FREINYES** [ppy/osu#18251] [Q&A] [answered]
Edition android will add vstroeniy administered to the mouse?

![Screenshot_2022-05-13-17-46-21-737_sh.ppy.osulazer.jpg](https://user-images.githubusercontent.com/104691904/168309696-9042b03f-67c5-415c-8a21-74f533a86808.jpg)![Screenshot_2022-05-13-17-46-06-160_sh.ppy.osulazer.jpg](https://user-images.githubusercontent.com/104691904/168309710-de0722aa-5300-4153-81cf-b63cfdd705cc.jpg)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/16464
